biojava-structure/src/main/java/org/biojava/nbio/structure/cluster/SubunitClusterUtils.java

```diff
@@ -65,7 +65,8 @@ public static String getStoichiometryString(List<SubunitCluster> clusters) {
 	public static boolean isPseudoStoichiometric(List<SubunitCluster> clusters) {
 
 		for (SubunitCluster c : clusters) {
-			if (c.getClustererMethod() == SubunitClustererMethod.STRUCTURE)
+			if (c.getClustererMethod() == SubunitClustererMethod.STRUCTURE ||
+				c.getClustererMethod() == SubunitClustererMethod.SEQUENCE)
```

Comment:
This will cause assemblies with different entity chains (not 100% identical) to be called pseudo-stoichiometric. I am not saying this is a bad thing, just in case you were not aware.

Reply:
To me any non-identical sequence should be consider pseudo. Perhaps I'd lower the identity theshold to 95% (instead of 100%). Would it be possible to get the threshold here?

Reply:
The Subunit clustering algorithm is designed with three levels:

- Identity: chains with same SEQRES (same entity) are clustered together
- Sequence: sequence alignment is performed on the SEQRES and if the sequence identity is higher than a threshold chains are clustered together
- Structure: structure alignment with a RMSD and coverage threshold

I think there is not a unique definition of pseudo-stoichiometry, so maybe we should give a clear definition for this flag. In the context of symmetry analysis, I would say that structural alignment has to be used to generate the pseudo-stoichiometric alignment (geometric property), so this was the reason to set TRUE only for structural clustering.

Reply:
Since the user can change the sequence identity threshold, I agree that a sequence similarity clustering with 30% threshold should be considered pseudo-stoichiometry, for example.

Reply:
I might have complicated things too much. Maybe the easiest solution is to just re-implement the first level (Identity) to be a sequence alignment at fixed 95% sequence identity.

Reply:
>Maybe the easiest solution is to just re-implement the first level (Identity) to be a sequence alignment at fixed 95% sequence identity.

I've added a commit that does exactly that. Could you recheck if it looks ok?